DataManagementSystem/DB/FileCatalogComponents/DatasetManager.py

```diff
@@ -19,7 +16,7 @@
 
 class DatasetManager:
```

Comment:
There is a doc string in the module. I think for the main class of the module this should be rather the module documentation